src/Agent.Worker/ExecutionContext.cs

```diff
@@ -19,13 +19,14 @@ public interface IExecutionContext : IAgentService
         TaskResult? Result { get; set; }
         TaskResult? CommandResult { get; set; }
         CancellationToken CancellationToken { get; }
+        Int32 TimeoutInMinutes { get; }
```

Comment:
Job timeout is in mintues so to be consistent with job timeout, I kept task timeout in minutes.